macros/optimize_tables.sql

```diff
@@ -27,8 +27,12 @@ OPTIMIZE safe_ethereum.safes;
 OPTIMIZE seaport_ethereum.transfers;
 {% endset %}
 
-{% set sudoswap_ethereum_events %}
+{% set sudoswap_ethereum %}
 OPTIMIZE sudoswap_ethereum.events;
+OPTIMIZE sudoswap_ethereum.pool_creations;
```

Comment:
i wish this worked, but i'm being told from the team that each set has to include only one optimize statement, which is why all others in the file are set that way :(

Reply:
Ah, makes sense since you can only execute 1 statement in a do_run.